jenkins/master/Dockerfile

```diff
@@ -20,10 +20,11 @@ RUN import_certs.sh
 COPY plugins.txt /opt/openshift/configuration/plugins.txt
 COPY kube-slave-common.sh /usr/local/bin/kube-slave-common.sh
```

Comment:
@michaelsauter  . is there a reason for having this actually still in? ... just asking :)

Reply:
Probably not. I created a separate issue for this: https://github.com/opendevstack/ods-core/issues/863